src/sentry/static/sentry/app/views/organizationDiscover/conditions/utils.jsx

```diff
@@ -83,6 +83,11 @@ export function getExternal(internal, columns) {
       external[2] = internal.replace(strStart, '');
     }
 
+    // Ignore double quotes if they have been entered
+    if (external[2] !== null && external[2].match(/^".*"$/)) {
```

Comment:
What about enclosing single quotes?

Reply:
I thought about supporting single quotes this but decided not to go ahead now since it hasn't come up anecdotally yet and our other search fields like stream only support double quotes